app/Filament/Resources/ApplicationResource.php

```diff
@@ -161,6 +162,11 @@ public static function table(Table $table): Table
                     }),
                 Tables\Columns\TextColumn::make('display_name')
                     ->searchable(),
+                Tables\Columns\TextColumn::make('regstatus')
+                    ->getStateUsing(function (Application $record) {
+                        $reg = RegSysClientController::getSingleReg($record->user()->first()->reg_id);
```

Comment:
Won't requesting the status per record result in a significant impact on page load time?

Reply:
Potentially on prod, agreed. Disabled it for now and added a TODO.